spyder_profiler/__init__.py

```diff
@@ -3,4 +3,8 @@
 #==============================================================================
 # The following statement is required to register this 3rd party plugin:
 #==============================================================================
+from spyder.utils.qthelpers import qapplication
+# The line below is needed in order to prevent a segmentation fault.
+# See issue #4014
+app = qapplication()
```

Comment:
Its different. Cause we are left with an application lying around. When spyder is running, tat application object has been defined already.

If help is trying to instrospect these modules then it will create an Application, otherwise it wont create anything.

qapplication will always return an application whether it exists or not.